src/dynamics/time_integration.jl

```diff
@@ -119,6 +122,12 @@ function initialize!(L::Leapfrog, model::AbstractModel)
     if nΔt != output_dt
         @warn "$n steps of Δt = $(L.Δt_millisec.value)ms yield output every $(nΔt.value)ms (=$(nΔt.value/1000)s), but output_dt = $(output_dt.value)s"
     end
+
+    # reset to start with Euler step because
+    # simulation = initialize! also reallocates the prognostic variables
+    # by default without setting the 2nd step
+    L.start_with_euler = true
```

Comment:
the start_with_euler is always true, except for when the restart without reinialization is chosen?